projects/plugins/jetpack/extensions/blocks/recurring-payments/block.json

```diff
@@ -65,5 +65,5 @@
 			"type": "string"
 		}
 	},
-	"editorScript": "file:../editor.js"
+	"parent": [ "jetpack/payment-buttons" ]
```

Comment:
This block is currently broken for me:

<img width="822" alt="image" src="https://github.com/Automattic/jetpack/assets/426388/a1e683cd-6504-4a53-a405-4e801040ec44">

Reply:
I also noticed this in console:
<img width="860" alt="Screenshot at Oct 24 10-20-45" src="https://github.com/Automattic/jetpack/assets/60262784/f0c4805a-ba3b-4ab3-b1fc-f610a3d90e81">